Xamarin.Forms.Platform.Android/Renderers/MediaElementRenderer.cs

```diff
@@ -0,0 +1,310 @@
+using Android.Content;
+using Android.Media;
+using Android.Views;
+using Android.Widget;
+using System;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+using Android.Graphics;
+using Android.Runtime;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public sealed class MediaElementRenderer : ViewRenderer<MediaElement, FrameLayout>, MediaPlayer.IOnCompletionListener, MediaPlayer.IOnPreparedListener
```

Comment:
I am wondering if we need this one at all? What is the benefit of the non-fast renderer? We just add an extra view to the UI and more code to maintain. Aren't we preferring the fast renderers all over?

Reply:
I'm not sure, but the platform code contains "slow" renderers for all the fast renderers so I'm assuming there are some cases it will fall back to using these? I'd rather we didn't need to maintain both of course!

Reply:
Where are those default interface members when we need them, eh? https://github.com/dotnet/csharplang/issues/52